src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -122,7 +122,7 @@ function ReportPreview({
     const [isPaidAnimationRunning, setIsPaidAnimationRunning] = useState(false);
     const [isHoldMenuVisible, setIsHoldMenuVisible] = useState(false);
     const [requestType, setRequestType] = useState<ActionHandledType>();
-    const [nonHeldAmount, fullAmount] = ReportUtils.getNonHeldAndFullAmount(iouReport, policy);
+    const {nonHeldAmount, fullAmount, isNonHeldAmountNegative} = ReportUtils.getNonHeldAndFullAmount(iouReport, policy);
```

Comment:
@mkzie2 Please rename `isNonHeldAmountNegative` to `hasValidNonHeldAmount`, as this makes more sense in the contexts where the boolean value is being checked.

`hasValidNonHeldAmount` should be `true` if the `nonHeldAmount` value is not negative and is greater than `0`.

Reply:
Thanks for the great suggestion. I updated.